bundle.yaml

```diff
@@ -15207,8 +15207,16 @@ spec:
                   changed after creation
                 type: integer
               databaseSecretName:
-                description: The name of the K8s secret that holds the password to the
-                  database.
+                description: The name of the secret that holds the password
+                  to the database (redis databases only).
+                  If secret does not exist, it will be created.
+                  To define the password, create an opaque secret and set the name in the spec.
+                  The password will be taken from the value of the 'password' key.
```

Comment:
this is addition after the release and it is fine, clarification for the doc only.